GTG/gtk/editor/taskview.py

```diff
@@ -963,3 +961,20 @@ def format_update(cls, text: str) -> str:
         text = text.replace('→', '')
 
         return text
+
+    def __update_style(self, settings, *args):
+        prefer_dark_theme = settings.get_property("gtk-application-prefer-dark-theme")
+        manager = GtkSource.StyleSchemeManager().get_default()
+        scheme_ids = manager.get_scheme_ids()
+
+        if 'Adwaita' in scheme_ids:
+            scheme_name = 'Adwaita'
+        else:
+            scheme_name = 'classic'
+
+        if prefer_dark_theme:
+            scheme = manager.get_scheme(f'{scheme_name}-dark')
+        else:
+            scheme = manager.get_scheme(scheme_name)
```

Comment:
Wouldn't it be simpler to write this like so -
```python
        scheme_name = 'Adwaita' if 'Adwaita' in scheme_ids else 'classic'
        scheme_name = f'{scheme_name}-dark' if prefer_dark_theme else scheme_name
        scheme = manager.get_scheme(scheme_name)
```

Or is this discouraged owing to possible incompatibility with older versions of Python?

Reply:
@diegogangl what are your thoughts on @subnut's suggestion?

Reply:
Not Diego, but my 2 cents: I don't have a religious preference against ternary operators, but I prefer to only use them when just testing a boolean.

For example, this is fine:
```
scheme_name = f'{scheme_name}-dark' if prefer_dark_theme else scheme_name
```

But this:
```
scheme_name = 'Adwaita' if 'Adwaita' in scheme_ids else 'classic'
```
is harder to parse. I'd do a double take that I wouldn't do with a normal if statement.

So I prefer the version that's currently written, but it's not a strong preference.

Reply:
I find the original 9-lines-long code much easier to read than the 3-lines version, and personally, code readability trumps compactness any time of the day. I personally would not have been able to easily understand the 3-lines version. In other words, [PEP-20](https://peps.python.org/pep-0020/)…

Reply:
I agree with @nekohayo. While @subnut's code is cleaner, it's better to keep it as readable as possible